packages/cli/src/util/telemetry/commands/list/index.ts

```diff
@@ -21,12 +21,15 @@ export class ListTelemetryClient extends TelemetryClient {
 
   trackCliOptionEnvironment(environment?: string) {
     if (environment) {
-      if (environment !== 'production' && environment !== 'preview') {
-        environment = this.redactedValue;
-      }
+      const redactUnknownEnvironment = (environment: string) => {
```

Comment:
I would define this at the top-level since it doesn't depend on any scoped variables